pkg/rekor/identity.go

```diff
@@ -283,10 +284,19 @@ func GetCheckpointIndex(logInfo *models.LogInfo, checkpoint *util.SignedCheckpoi
 }
 
 func IdentitySearch(startIndex int, endIndex int, rekorClient *client.Rekor, monitoredValues identity.MonitoredValues, outputIdentitiesFile string, idMetadataFile *string) ([]identity.MonitoredIdentity, error) {
-	entries, err := GetEntriesByIndexRange(context.Background(), rekorClient, startIndex, endIndex)
+	resp, err := util.Retry(context.Background(), func() (interface{}, error) {
+		entries, err := GetEntriesByIndexRange(context.Background(), rekorClient, startIndex, endIndex)
```

Comment:
Can we wrap each of the outbound calls in https://github.com/sigstore/rekor-monitor/blob/e08902beea4f2aa7ab9934f69d153dd7613e3b88/pkg/rekor/client.go#L52 rather than here? Otherwise, we'll retry a potentially large range of requests.